### PR TITLE
fam: Add FAM wrappers for vfio_irq_set

### PR DESCRIPTION
**Pull request by sameo**
Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>

### CODE REVIEW

**Comment by sameo** [comment]
@jiangliu This would remove the need for PR #3, and remove the need for having fam implementations in the vfio-ioctls crate.

**Comment by jiangliu** [comment]
> @jiangliu This would remove the need for PR #3, and remove the need for having fam implementations in the vfio-ioctls crate.

It's OK to dismiss PR #3, the real issue PR #3 tries to solve hard-coded version number "0.1.1" of vmm-sys-util. Glad to see that vfio-ioctls doesn't need to handle serde issues:)

File: src/lib.rs [hunk 1]
```diff
@@ -1,6 +1,13 @@
 // Copyright 2019 Intel Corporation. All Rights Reserved.
 // SPDX-License-Identifier: (BSD-3-Clause OR Apache-2.0)
 
+#[macro_use]
+#[cfg(feature = "fam-wrappers")]
```
Comment (jiangliu) [inline src/lib.rs]: Should we swap line 4 and line 5?

**Comment by sameo** [comment]
This one needs a little more refining, it does not seem to pass the VFIO tests when being run against Cloud Hypervisor.

**Comment by sameo** [comment]
Tested against cloud-hypervisor, it passes our VFIO tests, see https://github.com/cloud-hypervisor/cloud-hypervisor/pull/679

**Comment by sboeuf** [comment]
Let's merge it!!

**Comment by jiangliu** [comment]
@sameo I nominate myself as code reviewer/owner for vfio-bindings and vfio-ioctls repo, so we could move forward fast:)

**Pull request merged**